lib/src/hitmap.dart

```diff
@@ -40,8 +44,27 @@ class _HitInfo {
 /// Creates a single hitmap from a raw json object. Throws away all entries that
 /// are not resolvable.
 ///
+/// DEPRECATED: Migrate to createHitmapV2.
+///
 /// `jsonResult` is expected to be a List<Map<String, dynamic>>.
-Future<Map<String, HitMap>> createHitmap(
+Future<Map<String, Map<int, int>>> createHitmap(
+  List<Map<String, dynamic>> jsonResult, {
+  bool checkIgnoredLines = false,
+  String? packagesPath,
+}) async {
+  final result = await createHitmapV2(
+    jsonResult,
+    checkIgnoredLines: checkIgnoredLines,
+    packagesPath: packagesPath,
+  );
+  return result.map((key, value) => MapEntry(key, value.lineHits));
+}
+
+/// Creates a single hitmap from a raw json object. Throws away all entries that
```

Comment:
[nit] move the details to a new paragraph instead of immediately following the header sentence.

https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph